Dockerfile.rosbag

```diff
@@ -0,0 +1,48 @@
+FROM ros:galactic-ros1-bridge
+ENV DEBIAN_FRONTEND noninteractive
+
+# ROS
+ENV ROS1_DISTRO noetic
+ENV ROS2_DISTRO galactic
+
```

Comment:
This may be unnecessary.
See https://github.com/osrf/docker_images/blob/master/ros/galactic/ubuntu/focal/ros1-bridge/Dockerfile#L11-L12.